tests/util/test_linearizer.py

```diff
@@ -75,8 +80,10 @@ def test_linearizer_is_queued(self):
         self.assertFalse(linearizer.is_queued(key))
 
     def test_lots_of_queued_things(self):
-        # we have one slow thing, and lots of fast things queued up behind it.
-        # it should *not* explode the stack.
+        """Tests lots of fast things queued up behind a slow thing.
+
+        The stack should *not* explode when the fast thing completes.
```

Comment:
Should "the fast thing" be "the slow thing" here? Otherwise I'm not sure which of the fast things this refers to.

Reply:
Yep, that's a typo.